File: cpp/bc1036.cpp
#include <iostream>
#include <iomanip>
#include <cmath>

using namespace std;

int main()
{
	float valA, valB, valC, delta, r1, r2;
	
	cin >> valA >> valB >> valC;
	
	delta = pow(valB, 2) -4*valA*valC;
	
	if(valA == 0 || delta < 0) cout << "Impossivel calcular" << endl;
	else
	{
		r1 = ((valB - sqrt(delta)) / (2*valA))*-1;
		r2 = ((valB + sqrt(delta)) / (2*valA))*-1;
		
		cout << "R1 = " << fixed << setprecision(5) << r1 << endl;
		cout << "R2 = " << fixed << setprecision(5) << r2 << endl;
	}
}

File: python/bc1036.py
from math import pow, sqrt

cVal = input().split(" ")

valA = float(cVal[0])
valB = float(cVal[1])
valC = float(cVal[2])

delta = pow(valB, 2) -4*valA*valC

if delta < 0 or valA == 0.0:
    print("Impossivel calcular")
else:
    r1 = (-valB + sqrt(delta)) / (2.0*valA)
    r2 = (-valB - sqrt(delta)) / (2.0*valA)

    print(f"R1 = {r1:.5f}")
    print(f"R2 = {r2:.5f}")